src/Control/SessionHandler/CacheSessionHandler.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace SilverStripe\Control\SessionHandler;
+
+use SensitiveParameter;
+use Psr\SimpleCache\CacheInterface;
+
+/**
+ * Session save handler that stores session data in an in a PSR-16 cache adapter.
+ */
+class CacheSessionHandler extends AbstractSessionHandler
+{
+    private CacheInterface $cacheHandler;
```

Comment:
```suggestion
    private CacheInterface $cache;
```

Note sure what the 'Handler' suffix is for? Also update all the references to it